data/human/outfits.txt

```diff
@@ -395,6 +395,17 @@ outfit "Nerve Gas"
 	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
 
+outfit "Old Man's Sword"
+	category "Unique"
+	cost 220000
+	thumbnail "outfit/unknown"
+	"capture attack" 1.4
+	"capture defense" 1
+	"unplunderable" 1
+	description "Received this from a pair of siblings whom you helped out by fulfilling their eccentric father's last will and testament on Prime."
+	description "	While swords are seen today as being old curios and commemorative objects, rendered useless when gunpowder became dominant on the battlefield thousands of years ago, this sword is unlike any you have seen before. Despite how heavy it looks it is remarkably light, and there are inscriptions on the blade itself that don't appear to be like any human language."
```

Comment:
"Received this from..." sounds very odd as a starting line for an outfit description. That can be adjusted.

 I think the whole first chunk here can be omitted as you know where you got the sword from, the whole adventure is supposed to leave its mark on the player rather than telling them as an afterthought. You also know this through the logs.